viewer/apiCrons.js

```diff
@@ -242,7 +242,7 @@ class CronAPIs {
 
     let minTimestamp;
     try {
-      const { body: data } = await Db.getMinValue(['sessions2-*', 'sessions3-*'], '@timestamp');
+      const { body: data } = await Db.getMinValue(Db.defaultIndexPatterns(Config.getArray('queryExtraIndices', '')), '@timestamp');
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.